packages/discord.js/typings/index.d.ts

```diff
@@ -6961,78 +6944,78 @@ export interface WidgetChannel {
 }
 
 export interface WelcomeChannelData {
+  channel: AnnouncementChannel | ForumChannel | MediaChannel | Snowflake | TextChannel;
   description: string;
-  channel: TextChannel | AnnouncementChannel | ForumChannel | MediaChannel | Snowflake;
   emoji?: EmojiIdentifierResolvable;
 }
 
 export interface WelcomeScreenEditOptions {
-  enabled?: boolean;
   description?: string;
+  enabled?: boolean;
   welcomeChannels?: readonly WelcomeChannelData[];
 }
 
 export interface ClientApplicationEditOptions {
+  coverImage?: Base64Resolvable | BufferResolvable | null;
   customInstallURL?: string;
   description?: string;
-  roleConnectionsVerificationURL?: string;
-  installParams?: ClientApplicationInstallParams;
+  eventWebhooksStatus?: ApplicationWebhookEventStatus.Disabled | ApplicationWebhookEventStatus.Enabled;
+  eventWebhooksTypes?: readonly ApplicationWebhookEventType[];
+  eventWebhooksURL?: string;
   flags?: ApplicationFlagsResolvable;
-  icon?: BufferResolvable | Base64Resolvable | null;
-  coverImage?: BufferResolvable | Base64Resolvable | null;
+  icon?: Base64Resolvable | BufferResolvable | null;
+  installParams?: ClientApplicationInstallParams;
   interactionsEndpointURL?: string;
-  eventWebhooksURL?: string;
-  eventWebhooksStatus?: ApplicationWebhookEventStatus.Enabled | ApplicationWebhookEventStatus.Disabled;
-  eventWebhooksTypes?: readonly ApplicationWebhookEventType[];
+  roleConnectionsVerificationURL?: string;
   tags?: readonly string[];
 }
 
 export interface ClientApplicationInstallParams {
-  scopes: readonly OAuth2Scopes[];
   permissions: Readonly<PermissionsBitField>;
+  scopes: readonly OAuth2Scopes[];
 }
 
-export type Serialized<Value> = Value extends symbol | bigint | (() => any)
+export type Serialized<Value> = Value extends bigint | symbol | (() => any)
   ? never
-  : Value extends number | string | boolean | undefined
+  : Value extends boolean | number | string | undefined
     ? Value
     : Value extends JSONEncodable<infer JSONResult>
       ? JSONResult
-      : Value extends ReadonlyArray<infer ItemType>
+      : Value extends readonly (infer ItemType)[]
         ? Serialized<ItemType>[]
         : Value extends ReadonlyMap<unknown, unknown> | ReadonlySet<unknown>
           ? {}
           : { [K in keyof Value]: Serialized<Value[K]> };
 
-//#endregion
+// #endregion
 
-//#region Voice
+// #region Voice
 
 /**
- * @internal Use `DiscordGatewayAdapterLibraryMethods` from `@discordjs/voice` instead.
+ * @internal
```

Comment:
Maybe use a `@remark` or something like that for the text instead of getting rid of it entirely

Also for the one in `InternalDiscordGatewayAdapterImplementerMethods` and `InternalDiscordGatewayAdapterCreator`